Userland/Libraries/LibCompress/Deflate.cpp

```diff
@@ -208,7 +208,10 @@ ErrorOr<bool> DeflateDecompressor::CompressedBlock::try_read_more()
             m_decompressor.m_output_buffer.write({ &byte, sizeof(byte) });
         }
     } else {
-        TRY(m_decompressor.m_output_buffer.copy_from_seekback(distance, length));
+        auto copied_length = TRY(m_decompressor.m_output_buffer.copy_from_seekback(distance, length));
+
+        // TODO: What should we do if the output buffer is full?
+        VERIFY(copied_length == length);
```

Comment:
Might want to mention `LibCompress` in the commit title if you add this check.

Reply:
I wasn't sure whether to add it, the only reason why we are touching this here at all is to make sure that it stays as obviously broken as it was before (just in a different way, since earlier we would just loop infinitely).

Reply:
I go by: if I potentially affect something's behavior, it goes in the commit title